dataactcore/models/validationModels.py

```diff
@@ -28,8 +28,8 @@ class FileColumn(Base):
     __tablename__ = "file_columns"
 
     file_column_id = Column(Integer, primary_key=True)
-    file_id = Column(Integer, ForeignKey("file_type_validation.file_id"), nullable=True)
-    file = relationship("FileTypeValidation", uselist=False)
+    file_id = Column(Integer, ForeignKey("file_type.file_type_id", name="fk_file_column_file_type"), nullable=True)
```

Comment:
Good question...we discovered a while back that not putting in an explicit FK name was causing problems in alembic downgrades (I don't remember the specifics, and [this](http://alembic.zzzcomputing.com/en/latest/naming.html) implies that we should be okay without explicit constraint names, given our SQLAlchemy and Alembic versions, so ???)

It could well be something related to the way we're defining indices in the model, too--it looks like there are several ways to do that.

We haven't been consistent in constraint naming. Would be happy to do so going forward, but have also learned from experience that it's hard to make a constraint naming convention stick and not always worth the effort.